src/main/java/org/sert2521/gamename/Drivetrain/commands/DriveStraight.kt

```diff
@@ -0,0 +1,22 @@
+package org.sert2521.gamename.Drivetrain.commands
+
+import org.strongback.command.Command
+import org.strongback.drive.TankDrive
+
+/**
+ * This command will drive at a constant speed for a set duration of time.
+ */
+class DriveStraight(val drive: TankDrive, val speed: Double, duration: Double) : Command(duration, drive) {
+    override fun execute(): Boolean {
+        this.drive.tank(speed, speed)
+        return false
+    }
+
+    override fun interrupted() {
+        this.drive.stop()
+    }
+
+    override fun end() {
+        this.drive.stop()
```

Comment:
You should be able to `Alt + Enter` on the method name and "Convert to expression body" or something like that. Basically turn it into `override fun end() = drive.stop()`. I've never tried that when overriding methods so we'll see if it works.